src/components/BlockSpreadsheets/UserData.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const UserData = () => (
```

Comment:
Where's the actual `UserData`? I see just column headers here.

Reply:
The component name is bad. I'll change to `UserTable`